.github/workflows/build.yml

```diff
@@ -34,14 +26,14 @@ jobs:
     env: 
         MC_VERSION: ${{ matrix.mc_version }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: actions/setup-python@v3.0.0
         with:
           python-version: '3.x' # Version range or exact version of a Python version to use, using SemVer's version range syntax
           architecture: 'x64'
-      - uses: actions/setup-java@v2
+      - uses: actions/setup-java@v3
         with:
-          distribution: 'adopt-openj9' # See 'Supported distributions' for available options
```

Comment:
Adoptopenjdk temurin is still hotspot. If you go onto the readme for the ga plugin, it says adopt-openj9 is how you reference Semeru (which is what it got rebranded to, in contrast to the hotspot build which got renamed to temurin)

Reply:
What "ga plugin"?

[OpenJ9 has not been rebranded to Semeru.](https://www.eclipse.org/openj9/) That is [IBM's distribution of it.](https://developer.ibm.com/blogs/ibm-semeru-runtimes-now-available-from-eclipse-adoptium-marketplace/)
OpenJ9 is now directly [from Eclipse](https://github.com/eclipse-openj9) instead of Adopt and is still updated by itself.

Reply:
> [OpenJ9 has not been rebranded to Semeru.](https://www.eclipse.org/openj9/) That is [IBM's distribution of it.](https://developer.ibm.com/blogs/ibm-semeru-runtimes-now-available-from-eclipse-adoptium-marketplace/)
Ah,  so the IBM builds replaced AdoptOpenJDKS?
> The ["ga plugin" README](https://github.com/actions/setup-java#supported-version-syntax) still lists OpenJ9 as an Adopt distribution. This makes me wonder if they know that OpenJ9 is now hosted [by Eclipse](https://github.com/eclipse-openj9) instead of Adopt.
OpenJ9 has been under the eclipse foundation since 2017, I think just that in the move of AdoptOpenJDK to Eclipse they switched to the IBM build to distribute and just dropped their own?